Kudu.FunctionalTests/Jobs/WebJobsTests.cs

```diff
@@ -285,6 +288,95 @@ public void TriggeredJobTriggers()
         }
 
         [Fact]
+        public void ScheduledTriggeredJobTriggers()
+        {
+            RunScenario("ScheduledTriggeredJobTriggers", appManager =>
+            {
+                const string jobName = "job1";
+
+                TestTracer.Trace("Copying the script to the triggered job directory");
+
+                appManager.JobsManager.CreateTriggeredJobAsync(jobName, "run.cmd", JobScript).Wait();
+
+                // Wait for file watcher to pick up
+                Thread.Sleep(30 * 1000);
```

Comment:
I think its fine, it is better to have a test than not have a test even at the cost of time